Source/missiles.cpp

```diff
@@ -2645,7 +2654,7 @@ void AddGolem(int mi, Point src, Point dst, int /*midir*/, int8_t /*mienemy*/, i
 	Missiles[mi]._miVar2 = src.y;
 	Missiles[mi]._miVar4 = dst.x;
 	Missiles[mi]._miVar5 = dst.y;
-	if ((Monsters[id].position.tile.x != 1 || Monsters[id].position.tile.y != 0) && id == MyPlayerId)
+	if (Monsters[id].position.tile != Point { 1, 0 } && id == MyPlayerId)

```

Comment:
This is crying out for a wrapper method explaining what a golem at position 1,0 means 😂.

Reply:
`GolemHoldingCell`